packages/govuk-frontend/src/govuk/components/task-list/task-list.yaml

```
@@ -41,21 +41,9 @@ params:
         params:
           - name: tag
             type: object
+            isComponent: true
             required: false
-            description: Object containing the options for a tag that acts as the status for the task.
-            params:
-              - name: text
-                type: string
-                required: true
-                description: Text to use within the tag. If `html` is provided, the `text` argument will be ignored.
-              - name: html
-                type: string
-                required: true
-                description: HTML to use within the tag. If `html` is provided, the `text` argument will be ignored.
-              - name: classes
-                type: string
-                required: false
-                description: Classes to add to the tag.
+            description: Options for the tag component, if the status should use a tag.
```

Nice spot. Do you mind adding this to the macro options style guide Trello?

I've only checked briefly, but other `required: false` components don't qualify if they should be used

```suggestion
            description: Options for the tag component.
```

---

> Nice spot. Do you mind adding this to the macro options style guide Trello?

https://trello.com/c/smPF3ePN/18-nested-components-components-used-by-other-components-should-not-have-their-params-listed-in-the-top-level-component-instead-sho